tests/acceptance/features/bootstrap/WebDavLockingContext.php

```diff
@@ -839,6 +885,34 @@ public function waitForCertainSecondsToExpireTheLock(int $time): void {
 		\sleep($time);
 	}
 
+	/**
+	 * @Then :count locks should be reported for file :file inside the space :space of user :user by the WebDAV API
+	 *
+	 * @param int $count
+	 * @param string $file
+	 * @param string $spaceName
+	 * @param string $user
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 */
+	public function numberOfLockShouldBeReportedInProjectSpace(int $count, string $file, string $spaceName, string $user) {
+		$this->spacesContext->userSendsPropfindRequestToSpace($user, $spaceName, $file);
+		$this->featureContext->theHTTPStatusCodeShouldBe(207, "");
+		$responseXml = $this->featureContext->getResponseXml();
```

Comment:
I would suggest this flow here as well
```suggestion
		$response = $this->spacesContext->userSendsPropfindRequestToSpace($user, $spaceName, $file);
		$this->featureContext->theHTTPStatusCodeShouldBe(207, "", $response);
		$responseXml = $this->featureContext->getResponseXml($response);
```

Reply:
For this I'll make a follow up refactoring PR as the function `userSendsPropfindRequestToSpace` is used by many other functions so will be a lot of unrelated changes for this PR